hs_swat_modelinstance/models.py

```diff
@@ -230,29 +129,31 @@ class ModelParameter(AbstractMetaDataElement):
     def __unicode__(self):
```

Comment:
@mohamedmorsy See my comment from above.  (Also, you are missing a "return" statement)